nio_cli/commands/add.py

```diff
@@ -34,7 +34,7 @@ def run(self):
                 for file_name in files:
                     if file_name == 'requirements.txt':
                         reqs = os.path.join(root, file_name)
-                        subprocess.call([sys.executable, '-m', 'pip', 'install', '-r', reqs, '--user'])
+                        subprocess.call([sys.executable, '-m', 'pip', 'install', '-r', reqs])
```

Comment:
the `--user` flag was throwing errors in virtual environments and preventing installs from happening. I figure it is best to remove this for now until we can standardize how how to install dependencies for nio .